encoding/thrift/options.go

```diff
@@ -154,3 +156,21 @@ func (p protocolOption) applyRegisterOption(c *registerConfig) {
 func Protocol(p protocol.Protocol) Option {
 	return protocolOption{Protocol: p}
 }
+
+// NoWire is an option that specifies to *not* use the thriftrw.Wire
+// intermediary format.  Note that if this is enabled, and a
+// 'protocol.Protocol' is provided (e.g., via thrift.Protocol)m that provided
+// protocol must be able to satisfy the 'stream.Protocol' interface.
```

Comment:
outdated comment. Now it's stream.RequestReader.
```suggestion
// protocol must be able to satisfy the 'stream.RequestReader' interface.
```